src/compas_rhino/install_with_pip.py

```diff
@@ -1,20 +1,117 @@
+import pathlib
+import random
+import string
 import subprocess
 
-import compas_rhino
+import click
 
+rhinocode = pathlib.Path().home() / ".rhinocode"
+rhinopython = rhinocode / "py39-rh8" / "python3.9"
+site_envs = rhinocode / "py39-rh8" / "site-envs"
 
-def install(args):
-    subprocess.check_call([compas_rhino._get_default_rhino_cpython_path("8.0"), "-m", "pip", "install"] + args)
 
+def random_string(length=8) -> str:
+    return "".join(random.choices(string.ascii_letters + string.digits, k=length))
```

Comment:
Interesting version of random string, I would have done uuid hex and crop to length but this also works